lib/features/family/features/missions/presentation/pages/missions_screen.dart

```diff
@@ -44,7 +44,7 @@ class _MissionsScreenState extends State<MissionsScreen> {
 
   @override
   Widget build(BuildContext context) {
-    const options = ['To do', 'Completed'];
+    final options = ['To do', context.l10n.completedKey];
```

Comment:
Just the completed, not the To do?

Reply:
I did a replaceall on 'completed' :) 
Todo can also be done now or as part of the translate missions item (KIDS-2186)